controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java

```diff
@@ -38,6 +41,7 @@
     public static final String deploymentAverageDuration = "deployment.averageDuration";
     public static final String deploymentFailingUpgrades = "deployment.failingUpgrades";
     public static final String deploymentBuildAgeSeconds = "deployment.buildAgeSeconds";
+    public static final String deploymentWarnings = "deployment.warnings";
```

Comment:
These constants should have had uppercase names to improve code readability. Not sure if a bad conventions should be continued or broken though. But nothing here is public api, so it should be safe to fix.